### DISCUSSION TITLE
Getting prosemirror nodes to use with Yjs

### DISCUSSION THREAD

**Discussion by addorai** [outline/rich-markdown-editor#484] [Q&A]
We're trying to build collaborative editing with Yjs and to load existing data Yjs [expects ProseMirror JSON as input](https://github.com/yjs/y-prosemirror). Is there a way to get the ProseMirror JSON via rich markdown editor?

**Comment by tommoor** [comment]
You don't need to get it, the y-prosemirror extension acts as a Prosemirror plugin so it bypasses the `defaultValue` prop.

Note, this project will be unmaintained soon: https://github.com/outline/rich-markdown-editor/discussions/481
